packages/flutter/lib/src/cupertino/date_picker.dart

```diff
@@ -1181,8 +1178,7 @@ class _CupertinoTimerPickerState extends State<CupertinoTimerPicker> {
               ),
             ),
           );
-        }
-        else
+        } else
```

Comment:
this one should have an opening curly too

Reply:
done